deployment/src/test/resources/application-cxf-test.properties

```diff
@@ -1,2 +1,9 @@
-quarkus.cxf.endpoint."/fruit".client-endpoint-url=http://localhost:8081/
-quarkus.cxf.endpoint."/fruit".service-interface=io.quarkiverse.cxf.deployment.test.FruitWebService
\ No newline at end of file
+
+
+quarkus.cxf.client."fruitclient".client-endpoint-url=http://localhost:8081/fruit
```

Comment:
Not happy about loosing compatibility of configuration but I have to think if it is better. Endpoint is more for server so make more sens to split client. And avoid the concatenation or URL with relative path. But still not sure. I have to think to it. Need pros cons

Reply:
PROS:

1. The complete URL is given in configuration, there is no need to "concat". This makes it easy for searching, i.e. which config is used talking to a service endpoint.

2. Many different client configurations possible.

Reply:
> The main con is see here is losing backward compatibility.

With some additional code it would be possible to soften this up allowing the old config as well. A warning could be given that this config style is subject to be disallowed. Nonetheless, given that version is 0.x, I guess that no-one is  expecting backward compatibility yet. Otherwise please raise your hands NOW!!